File: DE-CS545-WAA/Lab4/src/main/java/backend_design/lab4/book/domain/Book.java
package backend_design.lab4.book.domain;


import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.Getter;
import lombok.Setter;

@AllArgsConstructor
@Getter
@Setter
@Data
public class Book {
    private String isbn;
    private String author;
    private String title;
    private double price;
}

File: CS473DE/src/main/kotlin/homeworks/Book.kt
package homeworks

open class Book {
//    title author price
    var title:String
    var author:String
    var price: Double = 0.0

    constructor(title:String, author:String, price:Double) {
        this.title = title
        this.author = author
        this.price = price
    }

    open fun read() {
        println("Reading Paper book")
    }

}

